pandapower/networks/mv_oberrhein.json

```diff
@@ -514,17 +514,6 @@
       "is_multiindex": false,
       "is_multicolumn": false
     },
-    "characteristic": {

```

Comment:
Many json files that are used for test networks contained the definition of an empty 'characteristic' table. This refers to the old implementation where net.characteristic included the transformer spline characteristic objects. This table is not needed, that's why I removed it in this file because it was causing some pytests to fail.

Reply:
I have just updated all other json files that also defined an empty characteristic table by default to avoid any issues in the future (even though they were not introducing any failed tests right now). I did not update the json files in the old_versions folder because there are meant to maintain the old format.